src/Command/CheckUserCommand.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Command;
+
+use Doctrine\ORM\EntityManagerInterface;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+use App\Entity\Account;
+
+/**
+ * Command to convert a user into a Pelagos DRPM.
+ */
+class CheckUserCommand extends Command
+{
+    /**
+     * The Command name.
+     *
+     * @var string $defaultName
+     */
+    protected static $defaultName = 'pelagos:CheckUser';
+
+    /**
+     * A Doctrine ORM EntityManager instance.
+     *
+     * @var EntityManagerInterface $entityManager
+     */
+    protected $entityManager;
+
+    /**
+     * Class constructor for dependency injection.
+     *
+     * @param EntityManagerInterface $entityManager A Doctrine EntityManager.
+     */
+    public function __construct(EntityManagerInterface $entityManager)
+    {
+        $this->entityManager = $entityManager;
+        // It is required to call parent constructor if
+        // using a constructon in a Symfony command.
+        parent::__construct();
+    }
+
+    /**
+     * The command configuration.
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this->
+            setDescription('Check status of a user by username.')->
+            addArgument('username', InputArgument::REQUIRED, 'Username to be checked');
+    }
+
+    /**
+     * Symfony command execute section.
+     *
+     * @param InputInterface  $input  Required by Command.
+     * @param OutputInterface $output Required by Command.
+     *
+     * @throws \Exception If username not found.
+     *
+     * @return void
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
```

Comment:
Needs to return 0 (for no issues) or 1 (if there are any). Most of the old commands return void, I don't know why we did that, but it looks like doesn't break anything though

https://symfony.com/doc/4.4/console.html#creating-a-command

Reply:
I'll do what Mike had started doing, with the @return int Return code.